include/crpropa/advectionField/AdvectionField.h

```diff
@@ -44,38 +44,47 @@ class AdvectionFieldList: public AdvectionField {
 
 /**
  @class UniformAdvectionField
- @brief Advection field with one B-field vector.
+ @brief Advection field with one velocity/advection-field vector.
  */
 class UniformAdvectionField: public AdvectionField {
 	Vector3d value;
 public:
 	UniformAdvectionField(const Vector3d &value);
 	Vector3d getField(const Vector3d &position) const;
 	double getDivergence(const Vector3d &position) const;
+
+	std::string getDescription() const;
 };
 
 
 /**
 @class ConstantSphericalAdvectionField
-@brief Spherical advection with a constant wind speed
-
+@brief Spherical advection field with a constant wind speed
 */
 
 class ConstantSphericalAdvectionField: public AdvectionField {
 	Vector3d origin; //origin of the advection sphere
-	double vWind; // maximum wind velocity
+	double vWind; // wind velocity
 public:
-	ConstantSphericalAdvectionField(Vector3d origin, double vWind);
+	/** Constructor
+	 @param origin	Origin of the advection field
+	 @param vWind	Constant wind velocity
+
+*/
+
+	ConstantSphericalAdvectionField(const Vector3d origin, double vWind);
```

Comment:
I meant const reference, i.e. const Vector3& origin. However, in the constructor/setters this is only a minor thing only mentioned for consistency.